### PR TITLE
[ruby] Use new esbuild script

### PR DESCRIPTION
**Pull request by cb1kenobi**

### CODE REVIEW

**Review by styfle — CHANGES_REQUESTED** [review]
Same comments as the others

File: packages/ruby/build.mjs [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import { esbuild } from '../../utils/build.mjs';
```
Comment (TooTallNate) [inline packages/ruby/build.mjs]: Delete

File: .changeset/smooth-ravens-cheat.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+---
+---
```
Comment (TooTallNate) [inline .changeset/smooth-ravens-cheat.md:3]: Patch

File: packages/ruby/package.json [hunk 3]
```diff
@@ -15,7 +15,7 @@
     "directory": "packages/ruby"
   },
   "scripts": {
-    "build": "node build",
+    "build": "node build.mjs",
```
Comment (TooTallNate) [inline packages/ruby/package.json]:
```suggestion
    "build": "node ../../build-builder.mjs",
```

File: packages/ruby/tsconfig.json [hunk 4]
```diff
@@ -13,5 +13,7 @@
     "outDir": "dist",
     "strict": true,
     "target": "ES2021"
-  }
+  },
+  "include": ["src/**/*"],
+  "exclude": ["node_modules"]
```
Comment (TooTallNate) [inline packages/ruby/tsconfig.json]:
```suggestion
```

File: .changeset/smooth-ravens-cheat.md (hunk 2 above)
Comment (cb1kenobi) [reply .changeset/smooth-ravens-cheat.md:3]: Fixed

File: packages/ruby/tsconfig.json (hunk 4 above)
Comment (cb1kenobi) [reply packages/ruby/tsconfig.json]: Fixed

**Review by styfle — APPROVED** [review]

**Pull request merged**